BangazonAPI/Models/Computer.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+	public class Computer
+	{
```

Comment:
Hi! I think maybe there's supposed to be some properties in here-- are we missing some code?

Reply:
yep i added it all back in there  check now